extensions/amp-story/1.0/amp-story-cta-layer.js

```diff
@@ -47,17 +47,23 @@ export class AmpStoryCtaLayer extends AmpStoryBaseLayer {
   /** @override */
   buildCallback() {
     super.buildCallback();
-    this.setOrOverwriteTargetAttribute_();
+    this.setOrOverwriteAttributes_();
   }
 
   /**
-   * Overwrite or set target attribute to _blank in call-to-action links.
+   * Overwrite or set target attributes that are cta-layer-specific.
    * @private
    */
-  setOrOverwriteTargetAttribute_() {
+  setOrOverwriteAttributes_() {
     const ctaLinks = this.element.querySelectorAll('a');
     for (let i = 0; i < ctaLinks.length; i++) {
       ctaLinks[i].setAttribute('target', '_blank');
+      ctaLinks[i].setAttribute('role', 'link');
```

Comment:
I think we should only overwrite the `role` attribute when it was previously unset.  Otherwise LGTM